### ISSUE TITLE
VS code Install issue. "setup was unable to automatically close all applications..." and "Unable to delete code.exe"

### ISSUE THREAD

**Issue by aayushkamal** [microsoft/vscode#230997]
Does this issue occur when all extensions are disabled?: Ye

- VS Code Version: 1.94.1
- OS Version: win10

I am getting below error while upgrading my visual code.

![Image](https://github.com/user-attachments/assets/80340f0d-34f9-472e-81f9-9d24bf11024c)

Before this it was throwing this error
![Image](https://github.com/user-attachments/assets/a30a3ab0-3a79-43c3-a5dc-00efdc6b848f)
but after a restart now I am getting "Set up was unable" error.

can anyone please help ?

**Comment by gjsjohnmurray** [comment]
Maybe you previously installed from the System Installer (which will have requested elevated privileges) and now you are trying to upgrade from the User Installer?

**Comment by aayushkamal** [comment]
@gjsjohnmurray Yes, the previous version was installed using a system installer, and I am now also installing the new version using a system installer as an admin.

**Comment by gjsjohnmurray** [comment]
I suggest you restart Windows, then uninstall the old version, then install the new one.

**Comment by aayushkamal** [comment]
Thanks you @gjsjohnmurray 
Can you please help me on uninstallation part:

I can see below error in logs:
Failed to uninstall "Microsoft Visual Studio Code 1.94.1" - Exception calling "Delete" with "3" argument(s): "(5) Access is denied: [C:\Program Files\Microsoft VS Code\Code.exe]" (System.UnauthorizedAccessException)

In ARP entry I cannot see the visual code but the the folder still exist.
![Image](https://github.com/user-attachments/assets/345474ce-89a6-4ac0-9839-84d7e8e1aa48)

and If I retrigger the installation it is giving thing error:

![Image](https://github.com/user-attachments/assets/7e08d705-982b-4b04-8c07-8b157bd8c3b9)

**Comment by gjsjohnmurray** [comment]
Did you try uninstalling from an elevated account? And did you reboot Windows first?

**Comment by aayushkamal** [comment]
Yes, I rebooted the machine and I am using elevated account but it still does not remove "Microsoft visual code" folder.

**Comment by aayushkamal** [comment]
okay Thank you 
"Is there any other process that is preventing the system from terminating code.exe or using it? After clicking 'Try again,' it proceeded successfully. any idea how can I disable this popup or automate the 'Try again' response to continue the installation?"

**Comment by gjsjohnmurray** [comment]
> any idea how can I disable this popup or automate the 'Try again' response to continue the installation?

No, but how often are you expecting to do this?

**Comment by aayushkamal** [comment]
one time only but I have multiple machines
